tests/test_specials.py

```diff
@@ -101,3 +109,56 @@ def test_slash_h_alias(executor):
     h_results = executor('\h SELECT')
     results = executor('\? SELECT')
     assert results[3] == h_results[3]
+
+
+@contextmanager
+def temp_filepath(filename):
+    try:
```

Comment:
Hey @catherinedevlin, py.test provides a fixture that does this already: http://pytest.org/latest/tmpdir.html

:)